src/dump.c

```diff
@@ -832,7 +832,7 @@ static void jl_serialize_value_(ios_t *s, jl_value_t *v)
         writetag(s, jl_method_type);
         jl_method_t *m = (jl_method_t*)v;
         union jl_typemap_t *tf = &m->specializations;
-        if (tf->unknown && tf->unknown != jl_nothing) {
+        if (0 && tf->unknown && tf->unknown != jl_nothing) {
```

Comment:
oops, got some other wip tests mixed up in here. I was doing testing because it appeared that this tfunc prunning may actually be hurting more than helping now.